src/main/java/seedu/address/logic/parser/ViewMeetingCommandParser.java

```diff
@@ -17,12 +18,17 @@ public class ViewMeetingCommandParser implements Parser<ViewMeetingCommand> {
      * @throws ParseException if the user input does not conform the expected format
      */
     public ViewMeetingCommand parse(String args) throws ParseException {
+        ArgumentMultimap argumentMultimap = ArgumentTokenizer.tokenize(args, PREFIX_INDEX);
+
+        Index index;
         try {
-            Index index = ParserUtil.parseIndex(args);
-            return new ViewMeetingCommand(index);
+            index = ParserUtil.parseIndex(argumentMultimap.getValue(PREFIX_INDEX).get());
         } catch (ParseException pe) {
-            throw new ParseException(
-                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, ViewMeetingCommand.MESSAGE_USAGE), pe);
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    ViewMeetingCommand.MESSAGE_USAGE), pe);
         }
+
+        return new ViewMeetingCommand(index);
```

Comment:
Do check these two lines coverage. I checked on mine and the line coverage is 71%.

Reply:
Should be completed now, please review. Thank you 😄